src/style.css

```diff
@@ -1 +1,78 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+}
+
+.nav__link {
+  text-decoration: none;
+  display: block;
+  padding: 22.5px 0;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 4px;
+  background-color: #00acdc;
+}
+
+.nav__li {
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__li:last-child {
+  margin-right: 0;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  margin: 0 50px 0 auto;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0 0 50px;
```

Comment:
`align-tems: center` is much better option, what if we will decide to make header higher, or just make links higher? It's better to avoid values hardcoding